.github/workflows/robot-ui-e2e-tests.yml

```diff
@@ -206,30 +182,22 @@ jobs:
                     restore-keys: |
                         ${{ runner.os }}-pip-
 
-            -   name: Install packages
+            -   name: Install robotframework-suite-tests folder
                 run: |
-                    python3 -m pip install --upgrade pip
-                    python3 -m pip install -U robotframework
-                    python3 -m pip install -U robotframework-requests
-                    python3 -m pip install -U robotframework-jsonlibrary
-                    python3 -m pip install -U robotframework-databaselibrary
-                    python3 -m pip install -U robotframework-browser
-                    python3 -m pip install PyMySQL
-                    sudo npx playwright install-deps
+                    cd ./data && composer require "spryker/robotframework-suite-tests:dev-master" --dev --no-interaction
+                    cp -r vendor ../vendor
 
             -   name: Install docker-compose
                 run: |
                     sudo curl -L "https://github.com/docker/compose/releases/download/2.12.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
                     sudo chmod +x /usr/local/bin/docker-compose
-            -   name: Install Robot tests
-                run: |
-                    git clone https://github.com/spryker/robotframework-suite-tests.git --single-branch --branch master robotframework-tests
+
             -   name: Install Project
                 continue-on-error: false
                 run: |
                     git clone https://github.com/spryker/docker-sdk.git ./docker
                     docker/sdk boot deploy.ci.acceptance.mariadb.robot.yml
-                    sudo bash -c "echo '127.0.0.1 backend-api.eu.spryker.local backend-api.us.spryker.local backend-gateway.eu.spryker.local backend-gateway.us.spryker.local backoffice.eu.spryker.local backoffice.us.spryker.local glue-backend.eu.spryker.local glue-backend.us.spryker.local glue-storefront.eu.spryker.local glue-storefront.us.spryker.local glue.eu.spryker.local glue.us.spryker.local mail.spryker.local mp.eu.spryker.local mp.us.spryker.local queue.spryker.local scheduler.spryker.local spryker.local swagger.spryker.local yves.eu.spryker.local yves.us.spryker.local date-time-configurator-example.spryker.local' >> /etc/hosts"
+                    sudo bash -c "echo '127.0.0.1 backend-api.eu.spryker.local backend-api.us.spryker.local backend-gateway.eu.spryker.local backend-gateway.us.spryker.local backoffice.eu.spryker.local backoffice.us.spryker.local glue-backend.eu.spryker.local glue-backend.us.spryker.local glue-storefront.eu.spryker.local glue-storefront.us.spryker.local glue.eu.spryker.local glue.us.spryker.local mail.spryker.local mp.eu.spryker.local mp.us.spryker.local queue.spryker.local spryker.local swagger.spryker.local yves.eu.spryker.local yves.us.spryker.local date-time-configurator-example.spryker.local' >> /etc/hosts"
```

Comment:
I'm not sure we need MP endpoints here. B2C shop does not contain those modules.
Please check here and other places